t/02-rakudo/03-corekeys.t

```diff
@@ -794,6 +794,7 @@ my @allowed =
         Q{$¢},
         Q{&postcircumfix:<[; ]>},
         Q{&postcircumfix:<{; }>},
+        Q{&postcircumfix:<{ }>},
```

Comment:
I'm surprised this is needed?

Reply:
Without it the test fails. And there is a new symbol in CORE::v6e so it is not complaining wrongly.